native_client/dotnet/README.rst

```diff
@@ -28,7 +28,8 @@ Prerequisites
 -------------
 
 
-* Windows 10
+* Windows 10 (Linux or OSX without UWP and NET Framework support using .NET Core)
+* `.NET Core SDK <https://dotnet.microsoft.com/download/dotnet-core/3.1>`_
```

Comment:
@carlfm01 Can you cross-check https://github.com/mozilla/community-tc-config/blob/main/imagesets/deepspeech-win2012r2/bootstrap.ps1 and verify if we need to modify it?